src/cmd-buildextend-live

```diff
@@ -341,6 +342,13 @@ def generate_iso():
                 buf = newbuf
                 karg_area_end = re.search(r'(#+) COREOS_KARG_EMBED_AREA\n', buf)
                 if karg_area_end is not None:
+                    if len(default_kargs) == 0:
```

Comment:
Even on next iterations, we should sanity-check that the default kargs are equal in each site. Then, it'd probably be cleaner to move the writing to a file outside of the `os.walk` loop.

Reply:
done